applications/nrf5340_audio/src/audio/streamctrl.h

```diff
@@ -0,0 +1,61 @@
+/*
+ * Copyright (c) 2018 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+/** @file streamctrl.h
+ *  @brief Stream control - control transfer and processing of the audio streams
+ *
+ * This is the module responsible for "running the system" after initial setup.
+ * The module assumes that before _start() is called
+ * - hw has been initialized
```

Comment:
hw = HW

Reply:
Removed this entire block, as we do not document any other file like this